src/mod/applications/mod_conference/conference_event.c

```diff
@@ -741,7 +741,15 @@ void conference_event_send_rfc(conference_obj_t *conference)
 switch_status_t conference_event_add_data(conference_obj_t *conference, switch_event_t *event)
 {
 	switch_status_t status = SWITCH_STATUS_SUCCESS;
+	conference_member_t *member = NULL;
 
+	for (member = conference->members; member; member = member->next) {
+		if (member->session) {
+			switch_channel_t *channel = switch_core_session_get_channel(member->session);
+			switch_channel_event_set_basic_data(channel, event);
```

Comment:
What was the expectation here, for example what if we have 100 members in the conference? so the only first member will be sent as the channel data?

Reply:
This was to retrieve the custom variables we already set in `switch_telnyx_channel_event_set_basic_data` so Telapps could troubleshoot issues regarding conference maintenance event. Sample log of this change when pushed to DEV -> http://tel-apps-logger.service.ch1-dev.consul:4010/ui?search=v3%3Ab8QeHPh0tOY_a3ea91txQKhSl9X-nbFmBAdVa8k4VAfvphVeoK7sgQ

Getting the actual member that trigger the event will be cumbersome change which I think will require updating `conference_event_add_data` with the member arg everywhere its called.

Thinking of it, using the `switch_telnyx_channel_event_set_basic_data` directly here will suffice since its majorly the telnyx_* variables that are needed by Telapps

Reply:
Or do we only get the member that is currently the moderator of the conference? But that depends on call control whether they always set a member as moderator

Reply:
We can remove the condition that checks for the call_control var since member's joining the conference might not be from a cal_control channel

Reply:
We could also have Telappt o effect a change from their end to have every channel bridged to the conference have the call_control var set before joining the conference

Reply:
I don't see any issues with your proposal, just stating the possible scenario that may not generate the variable that they needed

Reply:
Thanks. I've informed Telapps on the ticket with the limitations of this change and the expected behaviour.